app/controllers/admins_controller.rb

```diff
@@ -0,0 +1,60 @@
+class AdminsController < ApplicationController
+  before_action :set_admin, only: [:edit, :update, :destroy]
+
+  # GET /admins
+  def index
+    @admins = Admin.all
+  end
+
+  # GET /admins/new
+  def new
+    @admin = Admin.new
+  end
+
+  # GET /admins/1/edit
+  def edit
+  end
+
+  # POST /admins
+  def create
+    @admin = Admin.new(admin_params)
+
+    respond_to do |format|
+      if @admin.save
+        format.html { redirect_to admins_path, notice: 'admin was successfully created.' }
+      else
+        format.html { render :new }
+      end
+    end
+  end
+
+  # PATCH/PUT /admins/1
+  def update
+    respond_to do |format|
+      if @admin.update(admin_params)
+        format.html { redirect_to admins_path, notice: 'admin was successfully updated.' }
+      else
+        format.html { render :edit }
+      end
+    end
+  end
+
+  # DELETE /admins/1
+  def destroy
+    @admin.destroy
+    respond_to do |format|
+      format.html { redirect_to admins_url, notice: 'admin was successfully destroyed.' }
```

Comment:
```suggestion
      format.html { redirect_to admins_url, notice: 'admin was successfully deleted.' }
```
destroyed is not language that is commonly used for deletion